release-notes/azure/v11.3.0.md

```diff
@@ -0,0 +1,22 @@
+## :zap:  Giant Swarm Release 11.3.0 for Azure :zap:
+
+This release replaces CoreOS Container Linux that is going [end of life](https://coreos.com/os/eol/) on 26th of May 2020 with Flatcar Container Linux. 
+
+**The creation of new clusters with this release works as usual. New clusters will contain Flatcar OS from the start.**
+
+Migration process requires replacing VMSS CoreOS image with Flatcar image which, [due to Azure limitations](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-upgrade-scale-set#create-time-properties), is not possible. In order to enable existing clusters to upgrade with this release we had to amend standard procedure with additional steps that replace old VMSS with a new one, ensuring all workload is moved. 
+
+Flatcar image on the contrary of the CoreOS has a licence that has to be accepted. Please look into the instructions prior to upgrading your clusters [here](https://github.com/giantswarm/releases/blob/master/announcements/11-3-0-azure-flatcar-licence-approval.md)
+
+### Warning
+Following steps will be undertaken during the upgrade procedure:
+1. Once you start the upgrade, we will be notified in order to be hands on during the procedure.
+2. Upgrade will start with master replacement. Here old master is stopped as we need a final state of the ETCD data for the migration. Then new VMSS master instance is created (which due to Azure recent performance with disk provisioning can take several tries, hence extend the upgrade time). 
```

Comment:
```suggestion
2. The upgrade will start by replacing the master. The old master will be stopped as we need the final state of the ETCD data to be frozen for the migration. Once this is done, the new VMSS for the master instance will be created (:information_source: Due to the recent Azure disk provisioning performance,  this can take several tries, hence extend the upgrade time). 
```